index.html

```diff
@@ -51,8 +51,8 @@ <h3>Multiparty online game with voice communications</h3>
                    connections to peers without generating a separate
                    configuration ("offer") for each connection prior to
                    establishment.
-   N03             The user agent must be able to impose a bandwidth
-                   limit across mesh endpoints.
+   N03             Congestion control must be able to manage audio quality
+                   and latency in a fair manner between multiple connections.
```

Comment:
Need to make the same change up in line 40.

Reply:
Fixed.